source/s3_request_messages.c

```diff
@@ -443,6 +477,53 @@ static int s_s3_message_util_add_content_range_header(
     return AWS_OP_SUCCESS;
 }
 
+/* Add a content-md5 header. */
+static int s_s3_message_util_add_content_md5_header(
+    struct aws_allocator *allocator,
+    struct aws_byte_buf *byte_buf,
+    struct aws_http_message *out_message) {
+
+    AWS_PRECONDITION(out_message);
+
+    // Compute MD5
```

Comment:
Trivial: could we do all c-style comments just to keep everything consistent? :)